scripts/generate_pip_deps_from_conda.py

```diff
@@ -117,7 +127,7 @@ def main(conda_fname, pip_fname):
         f"# This file is auto-generated by scripts/generate_pip_deps_from_conda.py, "
         "do not modify.\n# See that file for comments about the need/usage of each dependency.\n\n"
     )
-    pip_content = header + "\n".join(pip_deps) + "\n"
+    pip_content = header + "\n".join(sorted(pip_deps)) + "\n"
```

Comment:
Why does it have to be sorted?

Reply:
In the Windows env file I split the dependencies by absolutely necessary vs. dev/docs related. The function here then took them in the same order, but `requirements-dev.txt` is sorted by another step in the precommit..